locale/templates/settings.ftl

```diff
@@ -0,0 +1,399 @@
+
+# When the user cannot use a QR code.
+tfa-enter-secret-key = Enter this secret key into your authenticator app:
+
+tfa-enter-totp = Now enter the security code from the authentication app.
+tfa-input-enter-totp =
+ .label = Enter security code
+tfa-save-these-codes = Save these one-time use codes in a safe place for when
+  you don’t have your mobile device.
+
+tfa-enter-code-to-confirm = Please enter one of your recovery codes now to
+  confirm you’ve saved it. You’ll need a code if you lose your device and want
+  to access your account.
+tfa-enter-recovery-code =
+ .label = Enter a recovery code
+
+## Profile section
+
+porfile-heading = Profile
+profile-display-name =
+  .header = Display name
+profile-password =
+  .header = Password
+profile-password-created-date = Created { $date }
+profile-primary-email =
+  .header = Primary email
+
+## Security section of Setting
+
+security-heading = Security
+
+## Sub-section row Defaults
+
+row-defaults-action-add = Add
+row-defaults-action-change = Change
+row-defaults-action-disable = Disable
+row-defaults-status = None
+
+## Recovery key sub-section on main Settings page
+
+rk-enabld = Enabled
+rk-not-set = Not Set
+rk-action-create = Create
+rk-action-remove = Remove
+rk-cannot-refresh = Sorry, there was a problem refreshing the recovery key.
+rk-key-removed = Account recovery key removed.
+rk-cannot-remove-key = Your account recovery key could not be removed.
+rk-refresh-key = Refresh recovery key
+rk-content-explain = Restores your information when you forget your password.
+rk-content-reset-data = Why does resetting my password reset my data?
+rk-cannot-verify-session = Sorry, there was a problem verifying your session
+rk-remove-modal-heading = Remove recovery key?
+rk-remove-modal-content = In the event you reset your password, you won't be
+  able to use your recovery key to access your data. You can't undo this action.
+
+## Secondary email sub-section on main Settings page
+
+se-heading = Secondary email
+  .header = Secondary Email
+se-cannot-refresh-email = Sorry, there was a problem refreshing that email.
+se-cannot-resend-code = Sorry, there was a problem re-sending the verification code.
+se-set-primary-successful = { $email } is now your primary email.
+se-set-primary-error = Sorry, there was a problem changing your primary email.
+se-delete-email-successful = { $email } email successfully deleted.
+se-delete-email-error = Sorry, there was a problem deleting this email.
+se-verify-session = You'll need to verify your current session to perform this action.
+se-verify-session-error = Sorry, there was a problem verifying your session.
+# Button to remove the secondary email
+se-remove-email =
+  .title = Remove email
+# Button to refresh secondary email status
+se-refresh-email =
+  .title = Refresh email
+se-unverified = unverified
+se-resend-code = Verification needed. <button>Resend verification code</button>
+  if it's not in your email or spam.
```

Comment:
`your email or spam` probably needs to be reworded. Maybe `your inbox or spam folder`?